src/Resources/app/administration/src/component/nosto-job-info/index.js

```diff
@@ -46,5 +46,9 @@ Component.register('nosto-job-info', {
                 this.jobItem = jobItem;
             });
         },
+
+        formatDate(date) {
+            return date ? (new Date(date)).toLocaleString() : '';
+        },
```

Comment:
what about having some sort of utils file which would have this function for formatDate rather than having same functions in different places :)

Reply:
Agreed

Reply:
Created it as a mixin, because a normal utils file is not available within the template